### PR TITLE
docs(components): Update Screen documentation to reflect maverick changes

### PR DESCRIPTION
**Pull request by silasjmatson**

### CODE REVIEW

File: docs/Components-Screen.md [hunk 1]
```diff
@@ -86,6 +149,29 @@ The valid values for this prop are: `"handled"`, `"always"`,and `"never"`.
...
+```tsx
+<Screen preset="scroll" ScrollViewProps={{ contentContainerStyle: { padding: 10 } }}>
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
+```
+
+### `scrollEnabledToggleThreshold`
+
+The `scrollEnabledToggleThreshold` prop is an optional `number` that determines the threshold at which the `scrollEnabled` prop of the `ScrollView` is toggled. This only applies for the `auto` preset. The default value is `{ percent: 0.92 }`. You can pass a point value in lieu of a percentage, e.g. `{ point: 100 }` will enable scrolling when the scroll view height is greater than 100 points larger than the scroll view content height.
```
Comment (silasjmatson) [inline docs/Components-Screen.md:170]: I think this should be "when the scroll view height is less than 100 points larger than the scroll view content height". Greather than/less than is hard, I swear.

File: docs/Components-Screen.md [hunk 2]
```diff
@@ -13,69 +13,132 @@ This is a component that renders a screen. It is used to wrap your entire screen
...
   // ... content here ...
 </Screen>
-````
+```
 
-### `unsafe`
+### `SafeAreaViewProps`
 
-The `unsafe` optional prop is a boolean that determines if the `KeyboardAvoidingView` is rendered as a `View` or a `SafeAreaView`. It defaults to `false`.
+The `SafeAreaViewProps` prop is a `SafeAreaViewProps` object that is passed to the [`SafeAreaView`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaview).
```
Comment (jamonholmgren) [inline docs/Components-Screen.md:96]: Probably repetitive, simplifying

```suggestion
The `SafeAreaViewProps` prop is an object that is passed as props to the [`SafeAreaView`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaview).
```

File: docs/Components-Screen.md [hunk 3]
```diff
@@ -13,69 +13,132 @@ This is a component that renders a screen. It is used to wrap your entire screen
...
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
 ```
 
-### `keyboardOffset`
+### `StatusBarProps`
 
-The `keyboardOffset` optional prop is a number that determines the amount of space between the bottom of the screen and the bottom of the keyboard. It defaults to 'none', which is `0`. You can define a custom offset in `screen.presets.ts`.
+The `StatusBarProps` prop is a `StatusBarProps` object that is passed to the `expo-status-bar` [`StatusBar`](https://docs.expo.io/versions/latest/sdk/status-bar/) component.
```
Comment (jamonholmgren) [inline docs/Components-Screen.md:107]:
```suggestion
The `StatusBarProps` prop is an object that is passed as props to the `expo-status-bar` [`StatusBar`](https://docs.expo.io/versions/latest/sdk/status-bar/) component.
```

File: docs/Components-Screen.md [hunk 4]
```diff
@@ -13,69 +13,132 @@ This is a component that renders a screen. It is used to wrap your entire screen
...
-}
+<Screen StatusBarProps={{ animated: false }}>
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
 ```
 
+### `KeyboardAvoidingViewProps`
+
+The `KeyboardAvoidingViewProps` prop is a `KeyboardAvoidingViewProps` object that is passed to the [`KeyboardAvoidingView`](https://reactnative.dev/docs/keyboardavoidingview).
```
Comment (jamonholmgren) [inline docs/Components-Screen.md:118]:
```suggestion
The `KeyboardAvoidingViewProps` prop is an object that is passed as props to the [`KeyboardAvoidingView`](https://reactnative.dev/docs/keyboardavoidingview).
```

File: docs/Components-Screen.md [hunk 5]
```diff
@@ -86,6 +149,29 @@ The valid values for this prop are: `"handled"`, `"always"`,and `"never"`.
...
 ```tsx
 <Screen preset="scroll" keyboardShouldPersistTaps="never">
+  <Header headerTitle="screen" />
+  // ... content here ...
+</Screen>
+```
+
+### `ScrollViewProps`
+
+The `ScrollViewProps` prop is a `ScrollViewProps` object that is passed to the React Native [`ScrollView`](https://facebook.github.io/react-native/docs/scrollview.html) component. This only applies for the `scroll` preset.
```
Comment (jamonholmgren) [inline docs/Components-Screen.md:159]:
```suggestion
The `ScrollViewProps` prop is an object that is passed as props to the React Native [`ScrollView`](https://facebook.github.io/react-native/docs/scrollview.html) component. This only applies for the `scroll` preset.
```

**Review by jamonholmgren — APPROVED** [review]
Looks good, some minor nits. 👍

**Comment by jamonholmgren** [comment]
CI issues aren't this PR, good to merge.

**Pull request merged**